src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -227,31 +215,34 @@ public void periodic() {
 
     // for logging
 
+    @Log(name = "Ingest Proximity")
     public boolean getIngestProximity() {
```

Comment:
This method looks the same as `ingestSensorHasBallIn()`, we should probably remove one of them (same for `feederSensorHasBallIn()` and `getFeederProximity()`).